src/Apple/Provider.php

```diff
@@ -134,24 +137,23 @@ public static function verify($jwt)
         $publicKeys = JWK::parseKeySet($data);
         $kid = $token->headers()->get('kid');
 
-        if (isset($publicKeys[$kid])) {
-            $publicKey = openssl_pkey_get_details($publicKeys[$kid]->getKeyMaterial());
+        if (!isset($publicKeys[$kid])) {
```

Comment:
This `if` is inverted to allow shortcut throwing the exception instead of dealing with an entire indentation level for the good case.